phoenix-android/src/main/res/values-de/important_strings.xml

```diff
@@ -30,6 +30,9 @@
     <string name="notif_pending_settlement_title">Bitte öffnen Sie Phoenix</string>
     <string name="notif_pending_settlement_message">Eine eingehende Zahlung steht aus.</string>
 
+    <string name="notif_inflight_payment_title">Eine Zahlung steht an</string>
+    <string name="notif_inflight_payment_message">Starten Sie Phoenix, damit die Zahlung irgendwann abgeschlossen werden kann.</string>
```

Comment:
```suggestion
    <string name="notif_inflight_payment_message">Starten Sie Phoenix, damit die Zahlung abgeschlossen werden kann.</string>
```
Or replace the `"irgendwann"` with `"rechtzeitig"`

Reply:
Thanks @dluvian